copyright

```diff
@@ -72,13 +72,13 @@ License: CC-BY-SA-4.0
 
 Files:
  images/ship/hai?flea*
+ images/projectile/rail?slug*
 Copyright: Matthew Smestad
 License: CC-BY-SA-4.0
 
 Files:
  images/outfit/railgun*
  images/outfit/railslug*
- images/projectile/rail?slug*
```

Comment:
why did you remove the railgun slug? seems unrelated

Reply:
Made a mistake in previous PR, I didn't really touched the projectile sprite.